src/Fixtures/index.js

```diff
@@ -1,11 +1,892 @@
 import animationMenageATrois from './AnimationMenageATrois';
 import animationSquare from './AnimationSquare';
 
+export const types = {
+  FITNESS: 'fitness',
+  TECHNICAL: 'technical',
+};
+
+export const intensities = {
+  LOW: 'Low',
+  MODERATE: 'Moderate',
+  HIGH: 'High',
+};
+
+export const goals = {
+  DEFENSE: 'Defense',
+  HANDLING: 'Handling',
+  CUTTING: 'Cutting',
+  THROWING: 'Throwing',
+  COMMUNICATION: 'Communication',
+  CATCH: 'Catch',
+  LEGS: 'Legs',
+  UPPER: 'Upper',
+  FULLBODY: 'Full body',
+  CORE: 'CORE',
+  CONDITIONING: 'Conditioning',
+};
+
+export const levels = {
+  BEGINNER: 'Beginner',
+  INTERMEDIATE: 'Intermediate',
+  ADVANCED: 'Advanced',
+};
+
+export const seasonTimings = {
+  OFFSEASON: 'Off-season',
+  PRESEASON: 'Pre-season',
+  INSEASON: 'In-season',
+  ANYTIME: 'Anytime',
+};
+
+export const equipmentLabels = {
+  NOEQUIPMENT: 'No Equipment',
+  BASIC: 'Basic Equipment',
+  FULLEQUIPMENT: 'Full equipment',
+};
+
 export default {
+  favoritesDrill: [],
```

Comment:
`drillFavorites` or `favoriteDrills` might be a better name

Reply:
`favoriteDrills` looks better to me

Reply:
but we should remove it here, it should be added in the other pull request